### DISCUSSION TITLE
Location of the setting file

### DISCUSSION THREAD

**Discussion by davidlehub** [pawelsalawa/sqlitestudio#4807] [General]
Hi, just a quick tip: Location of the setting file is in (in case some else want to copy it to an other computer):

C:\Users\%USERNAME%\ AppData\Local\SalSoft\SQLiteStudio

Cheer

**Comment by pawelsalawa** [comment]
It is also documented at https://github.com/pawelsalawa/sqlitestudio/wiki/Configuration_directory_location - for all supported platforms.
